services/clientlog/pkg/service/service.go

```diff
@@ -103,7 +103,8 @@ func (cl *ClientlogService) processEvent(event events.Event) {
 
 		evType = "postprocessing-finished"
 		data = FileReadyEvent{
-			ItemID: storagespace.FormatResourceID(*info.GetId()),
+			ParentItemID: storagespace.FormatResourceID(*info.GetParentId()),
```

Comment:
Do files always have a parentId or do I need to guard this with a nil check?

Reply:
Yes files should always have a parentID. The `root` has none, but it won't be in the `FileReadyEvent`